src/components/svg-canvas-graph.tsx

```diff
@@ -85,7 +85,7 @@ const SvgCanvas = () => {
         const { x, y } = getMousePosition(e);
 
         if (mode === 'free' && active === node) {
-            selected
+            [...selected]
```

Comment:
Will there be some performance issues if we convert set to array on every pointer move event? `selected` won't change in every move and IMO it's better to just `selected.forEach(s => {if (graph.current.hasNode(s)) graph.current.updateNodeAttributes(...)});`.